### DISCUSSION TITLE
The plans for DAOstar (Season 1)

### DISCUSSION THREAD

**Discussion by thelastjosh** [metagov/daostar#53] [Announcements]
[Updated August 23, 2023]

The main projects in Season 1:

1. **DAOstar Regulatory Interoperability** (a.k.a. Millennium Falcon), led by Captain Joni Pirovich, focused on defining a regulatory API / interface so that we can get legal benefits, especially limited liability status. This will eventually transition into a working group on tax standards.
2. **DAOstar Grants Management**, led by Eugene Leventhal, Kevin Olsen, and Joshua Tan, working with Gitcoin, OpenQ, DaoLens, Station, kycDAO, Solana, and others to define a common grants system standard for DAOs and other Web3 entities (e.g. L1 foundations), timed with the recent Gitcoin protocol launch and a [Web3 Grants research project](https://twitter.com/bbeats1/status/1694322607271621001?s=46&t=R3vfL4DPnWCHLekbYkBJxg) led by Eugene.
3. **DAOstar SAFE for DAOs**, basically creating a smart contract version of the YC SAFE using fancy game theory to verify equivalence.
4. **DAOstar Governor Ecosystem Standards** with Tally, OpenZeppelin, ScopeLift, Nouns, Joke DAO.
5. **DAOstar Attestations** (a.k.a. Voyager), led by Mendes at Avenue, focused on building an interoperable (and minimal!) attestations system for DAOs and DAO tooling, compatible with SBTs, DIDs, verifiable credentials, and other attestation tooling.
6. **DAOstar Delegations**, working with folks at Karma, Agora, Tally, Uniswap, and Optimism to define a delegate registry and standard schemas for delegation!
7. **DAOstar Asia Roundtable**, led by Hazel @unfriendlyamendment , to develop the DAO ecosystem in Asia and foster new standards and public goods (likely a marketing survey).
8. **DAOstar & Optimism Collaboration**, led @amanwithwings and funded by a generous grant from Optimism, to help DAOs on Optimism adopt 4824 and related standards.
10. More infrastructure development, including a new frontend, better indexing, multichain support (i.e. we already have Cosmos DAOs getting indexed), deeper indexing (so displaying not just the URIs but also retaining the membership rolls), and historical data.
11. Actual community management led by @amanwithwings and @unfriendlyamendment !

**Comment by unfriendlyamendment** [comment]
> The main projects in Season 2:
> 
> 1. **DAOstar Regulatory Interoperability** (a.k.a. Millennium Falcon), led by Captain Joni Pirovich, focused on defining a regulatory API / interface so that we can get legal benefits, especially limited liability status. This will eventually transition into a working group on tax standards.
> 2. **DAOstar Grants Management**, working with Gitcoin, OpenQ, DaoLens, Station, kycDAO, Solana, and others to define a common grants system standard for DAOs and other Web3 entities (e.g. L1 foundations), timed with the recent Gitcoin protocol launch.
> 3. **DAOstar SAFE for DAOs**, basically creating a smart contract version of the YC SAFE using fancy game theory to verify equivalence.
> 4. **DAOstar Governor Ecosystem Standards** with Tally, OpenZeppelin, ScopeLift, Nouns, Joke DAO.
> 5. **DAOstar Attestations** (a.k.a. Voyager), led by Mendes at Avenue, focused on building an interoperable (and minimal!) attestations system for DAOs and DAO tooling, compatible with SBTs, DIDs, verifiable credentials, and other attestation tooling.
> 6. More infrastructure development, including a new frontend, better indexing, multichain support (i.e. we already have Cosmos DAOs getting indexed), deeper indexing (so displaying not just the URIs but also retaining the membership rolls), and historical data.
> 7. Actual community management led by @amanwithwings !

Hi Josh - this is actually Season 1 (we are calling the previous iteration Season 0)!

**Reply by unfriendlyamendment** [reply]
@thelastjosh thanks Josh! and not to forget our newly launched delegation WG - DAOstar Delegation Working Group led by Mahesh @Karma. Working on building an on-chain delegation registry
